packages/components/src/components/button/_mixins.scss

```diff
@@ -104,3 +104,10 @@
     fill: $icon-color;
   }
 }
+
+@mixin button-padding-large {
+  align-items: baseline;
+  padding-top: $spacing-04;
```

Comment:
`padding-top` should be `$spacing-05`/16px

Reply:
```suggestion
  padding-top: $spacing-05;
```